packages/components/tsconfig.json

```
@@ -3,7 +3,9 @@
     "compilerOptions": {
         "outDir": "./lib",
         "rootDir": "./src",
-        "baseUrl": "./"
+        "paths": {
+            "@deriv/*": ["../*/src"]
+        }
     },
-    "include": ["./src/**/*.ts", "./src/**/*.tsx", "globals.d.ts"]
+    "include": ["./src", "./src/**/*.ts", "./src/**/*.tsx"]
```

```suggestion
    "include": ["src"]
```
@jim-deriv Just including `src` should be enough, But I'm not sure why the previous code was like that 🤔

---

Sure, what's your conclusion? I remove the rest?

---

@jim-deriv Not sure, Maybe there was a reason behind it 🤔 Better to check with someone first 🙇🏻 

cc: @ali-hosseini-fs

---

Yeah

---

if we are including whole src folder whats the point of adding ts and tsx files right after that?
we are somehow including all of the files in src dir.
@jim-deriv can you compare the result for both approaches to see if they are identical or not?
i think they should be the same and if yess we can safely remove those two extra ones.

---

@ali-hosseini-fs,

I checked this with `appstore`, and it turns out we don't need them. The result was the same 

cc: @farzin-fs